src/masternodes/mn_rpc.cpp

```diff
@@ -1504,28 +1482,39 @@ UniValue poolToJSON(DCT_ID const& id, CPoolPair const& pool, CToken const& token
     poolObj.pushKV("idTokenB", pool.idTokenB.ToString());
 
     if (verbose) {
-        poolObj.pushKV("reserveA", ValueFromAmount(pool.reserveA));
-        poolObj.pushKV("reserveB", ValueFromAmount(pool.reserveB));
+        poolObj.pushKV("reserveA", ValueFromAmount(pool.reserveA, pool.tokenA.decimal));
+        poolObj.pushKV("reserveB", ValueFromAmount(pool.reserveB, pool.tokenB.decimal));
```

Comment:
Why reserve becomes decimal?

Reply:
Reserves are based on two token ids in pool pair.

Reply:
Yes, it is correct, I only saw the second parameter of the input.